### ISSUE TITLE
Systray GUI Segfaults on Linux

### ISSUE THREAD

**Issue by iwalton3** [iwalton3/jellyfin-mpv-shim#16]
**Describe the bug**
The system tray icon, provided by the `pystray` library is causing the application to segfault on Linux. It seems to happen only when the `python-mpv` player backend is used.

**To Reproduce**
1. Install the application on Linux with `pystray`, `tkinter`, and `libmpv1`. 
2. Cast media to the application and then stop casting.
3. Right click on the tray icon.
4. The icon disappears and the program crashes with `Segmentation Fault`.

**Expected behavior**
The menu for the application appears.

**Desktop (please complete the following information):**
 - OS: Debian Stable w/ `libmpv1` compiled from git master
 - Version: All versions since 1.2.0.

**Error Messages**
The only message that appears in the terminal (not the application log) is `Segmentation Fault`. Attempts to trace the bug by using [faulthandler](https://docs.python.org/3/library/faulthandler.html) did not reveal any clues. This bug [has affected](https://www.reddit.com/r/jellyfin/comments/eq6xmy/jellyfin_mpv_shim_v121_new_features_and_bug_fixes/fh72iqt/) at least one other user on two separate Linux distributions.

**Workarounds**
There are two known workarounds for this issue.
 - Disable the GUI and systray by setting the option `enable_gui` to false. This eliminates the source of the problem.
 - Use external MPV. Ensure you can run MPV using mpv and then set `mpv_ext` to true. I have not observed the bug while the player was in this state.
 - If you right click the tray icon before casting something, the issue does not occur.

**Development Considerations**
There are two ways that I could probably work around the issue, but I don't like either of them.
 - Find a new systray library. PyQT5 comes to mind, but that library changes a lot and it is a large dependency.
 - Isolate the systray in another process. If this is caused by some interaction between `libmpv1` and `pystray`, it won't happen through a process. This is a dirty fix, but I am already doing something like this for `tkinter` since it also needs a "primary thread".

**Comment by iwalton3** [comment]
I did an `ltrace` of the program. It looks like the segfault is caused by `pango_cairo_show_layout`. Both `mpv` and GNOME Appindicator seem to be causing similar library calls to happen, and they are probably conflicting.
```
libgtk-3.so.0->cairo_save(0x355a090, 1, 1, 0)                                       = 0
libgtk-3.so.0->pango_layout_get_extents(0x33c0ad0, 0, 0x7ffc3b22c720, 3)            = 0
libgtk-3.so.0->pango_extents_to_pixels(0x7ffc3b22c720, 0, 0x335b0d0, 2)             = 18
libgtk-3.so.0->pango_layout_get_line_count(0x33c0ad0, 0, 0, 0)                      = 1
libgtk-3.so.0->pango_layout_get_type(0x354b9e0, 0x355a090, 0x3326230, 3)            = 0x2ff36a0
libgtk-3.so.0->cairo_save(0x355a090, 0x355a090, 0x3373980, 3)                       = 0
libgtk-3.so.0->pango_layout_get_context(0x33c0ad0, 0, 0x29eea80, 0x30220c8)         = 0x358c580
libgtk-3.so.0->pango_context_get_matrix(0x358c580, 0, 0x29eea80, 0x30220c8)         = 0
libgtk-3.so.0->cairo_move_to(0x355a090, 0, 0x33542b0, 0x30220c8)                    = 0
libgtk-3.so.0->gdk_cairo_set_source_rgba(0x355a090, 0x342ce80, 0x33c0ad0, 0x30220c8) = 0
libgtk-3.so.0->pango_cairo_show_layout(0x355a090, 0x33c0ad0, 0, 0x7f790c66bde0 <no return ...>
--- SIGSEGV (Segmentation fault) ---
+++ killed by SIGSEGV +++
```
I definitely don't think there is a clean way to fix this, so I'm simply going to isolate `pystray` into another process.

**Comment by iwalton3** [comment]
This is fixed in version 1.3.7. Note that there is no Windows release as this is a Linux-only bugfix release.

**Comment by iwalton3** [comment]
https://gitlab.gnome.org/GNOME/gtk/issues/2430
